volontulo/templates/homepage.html

```diff
@@ -6,19 +6,19 @@
     <div class="row">
         <div class="col-xs-4 text-center">
             <div>
-                <img src="http://placehold.it/260x260?text=Załóż+konto" />
+                <img class="img-responsive center-block" src="http://placehold.it/260x260?text=Załóż+konto" />
```

Comment:
Not sure if center-block is correctly used. As name implies it's block class, not for inline elements. img is inline element so text-center class on parent div should be used.